includes/class-gp-notifications.php

```diff
@@ -421,6 +421,20 @@ public static function get_email_body( WP_Comment $comment, array $comment_meta
 				}
 			}
 		}
+		if ( ! empty( $comment_meta['reject_reason'][0] ) ) {
```

Comment:
```suggestion
		if ( ! empty( maybe_unserialize( $comment_meta['reject_reason'][0] ) ) ) {
```

The empty data is serialized `a:0:{}`, so you need to unserialize before check if it is empty or not. If you don't do this, the translator is going to receive a line with empty value.

![image](https://user-images.githubusercontent.com/1667814/187561613-6a43bb5d-3fab-4b8a-b871-bfdb0292c824.png)

Reply:
The `empty()` call won't avoids the warning this way. PHP first evaluates `maybe_unserialize( $comment_meta['reject_reason'][0] )` and already encounters the warning there since it might access an undefined array item. Then the empty check is performend but this avoids a warning since the warning was already generated on the inside function. Thus you need to separate out the check with an isset:

```suggestion
		if ( isset( $comment_meta['reject_reason'][0] ) && ! empty( maybe_unserialize( $comment_meta['reject_reason'][0] ) ) {
```